build.sbt

```diff
@@ -34,9 +34,12 @@ lazy val commonSettings = {
       "org.apache.daffodil" %% "daffodil-sapi" % daffodilVer,
       "org.apache.daffodil" %% "daffodil-runtime1" % daffodilVer
     ),
+    dependencyOverrides ++= Seq(
+      "org.apache.commons" % "commons-lang3" % "3.12.0"
+    ),
     licenses += ("Apache-2.0", new URL("https://www.apache.org/licenses/LICENSE-2.0.txt")),
     organization := "org.apache.daffodil",
-    scalaVersion := "2.12.13",
+    scalaVersion := "2.12.15",
```

Comment:
This change is, FYI, super important because the latest LTS Java version, 17, requires this version of the scala libraries or obscure errors occur. 

And there are good reasons to want Java 17, including that it is noticibly faster.